G.A.M.M.A/modpack_addons/G.A.M.M.A. Additional Storylines fixes/gamedata/configs/text/eng/st_ah_quest_ah.xml

```diff
@@ -687,13 +687,13 @@
         <text>A special case with loot for the Bartender.</text>
     </string>
     <string id="cs_anom_pribor_desc">
-        <text>Special device for studying anomalous fields, belonging to the group "Clear Sky.</text>
+        <text>Special device for studying anomalous fields, belonging to the group Clear Sky.</text>
```

Comment:
Special device for studying anomalous fields, belonging to the Clear Sky **faction.**